osu.Game/Screens/Play/HUD/DefaultHealthDisplay.cs

```diff
@@ -78,30 +78,39 @@ public Color4 GlowColour
 
         public DefaultHealthDisplay()
         {
-            Size = new Vector2(1, 5);
+            const float padding = 20;
+            const float bar_height = 5;
+
+            Size = new Vector2(1, bar_height + padding * 2);
             RelativeSizeAxes = Axes.X;
-            Margin = new MarginPadding { Top = 20 };
 
-            InternalChildren = new Drawable[]
+            InternalChild = new Container
             {
-                new Box
-                {
-                    RelativeSizeAxes = Axes.Both,
-                    Colour = Color4.Black,
-                },
-                fill = new Container
+                Padding = new MarginPadding { Vertical = padding },
```

Comment:
Applying the padding on both sides has a small side effect, namely that the mod icons (and playback controls, if in replay) get shifted down a tiny bit:

| master | this PR |
| :-: | :-: |
| ![osu_2023-01-26_20-46-03](https://user-images.githubusercontent.com/20418176/214935726-4e4d4034-5d23-4394-98ea-8c547c9da80c.jpg) | ![osu_2023-01-26_20-46-51](https://user-images.githubusercontent.com/20418176/214935716-f7d485a5-179b-4415-ae68-664e7a05e18d.jpg) |

I don't think it matters in the slightest since these controls weren't really aligned to anything properly anyways, but I will wait for a second approval and/or confirmation from @peppy that this is A-OK before clicking the button.

Reply:
I think that's okay for now, yeah. I think I prefer the slightly larger padding.